### DISCUSSION TITLE
Cant uninstall teleport

### DISCUSSION THREAD

**Discussion by Lukylix** [gravitational/teleport#28602] [Q&A]
Hi !  
I need some help with a peculiar situation.  
I recently uninstalled teleport from my system using "apt -y remove teleport" and removed the binary files.  
However, when I checked using "pgrep -f teleport," I discovered three lingering processes. Even after repeatedly killing them, they keep coming back. It's baffling!

I thought I had removed everything, but apparently, there are remnants of teleport that refuse to disappear. I'm not sure why this is happening.

I'm reaching out to the community for any insights or solutions to fully remove teleport from my system as the documentation didn't work for me.

**Comment by dgkirkwood** [comment]
Hi @Lukylix - Is systemd managing the Teleport process on your machine? 
You may want to check with `systemctl status teleport`. 

If so you will want to stop the service there - `sudo systemctl stop teleport`
You would also want to remove the associated service file which will be mentioned in the status output. My install on Ubuntu has it at `/lib/systemd/system/teleport.service`

**Reply by Lukylix** [reply]
Thank you for your reply !
Unfortunatly i dont got any teleport service.
The service got removed on the first uniinstall.
I have also tried to rerun the install script 
and then disbale the service without succes.

**Reply by Lukylix** [reply]
Output of ps -ef | grep [t]eleport
```bash
root@iloa:~# ps -ef | grep [t]eleport
100000   1078153 1078105  0 03:55 ?        00:00:00 /bin/sh -c sleep 1 && /bin/dumb-init teleport start -c /etc/teleport/teleport.yaml
100000   1078912 1078153  0 03:55 ?        00:00:00 /bin/dumb-init teleport start -c /etc/teleport/teleport.yaml
100000   1078913 1078912  0 03:55 ?        00:01:50 teleport start -c /etc/teleport/teleport.yaml
```

I dont get it:
When type teleport i got :
-bash: teleport: command not found
Then i checked for the the /bin/dumb-init but it is not present.

I am totaly lost;

**Reply by Lukylix** [reply]
Is it possible that those processes are docker processes ?

(I am trying to connect teleport container to the host system)

**Reply by dgkirkwood** [reply]
I would expect to see `dumb-init` used in containerised applications. 

Are there any containers being run on this system?

**Reply by Lukylix** [reply]
Yes this must be the issue.
Si i cant install teleport using the script as it detect my container ?
Is there any other way
